projects/can_api/utils.py

```diff
@@ -3,19 +3,20 @@ def get_rx_messages(subs, messages):
     Gets the received Messages from the list of messages, based on the subs
     dictionary.
     """
+    # Check to see if subs has a message not specified in the DBC
+    dbc_message_names = [dbc_message.name for dbc_message in messages]
+    for message in subs:
+        if message["name"] not in dbc_message_names:
+            raise ValueError(
+                f"Subscription message {message['name']} was not found in the dbc messages."
```

Comment:
You don't have to change this at all, just a cool Python thing you might like though is this guy :=, the [walrus operator](https://realpython.com/python-walrus-operator/), basically lets you assign and return a value at the same time, so you write something like:

```
if name := message["name"] not in dbc_message_names:
    raise ValueError(f'{name} not found')
```

Anyway just a fairly new python feature that I think is cool haha.